python/lsst/meas/base/plugins.py

```diff
@@ -68,6 +71,103 @@
                     TransformClass=bl.ApertureFluxTransform, executionOrder=BasePlugin.FLUX_ORDER)
 
 # --- Single-Frame Measurement Plugins ---
+class SingleFrameVarianceConfig(SingleFramePluginConfig):
+        scale = lsst.pex.config.Field(dtype=float, default=5.0, optional=True,
+                                      doc="scale factor to apply to shape for aperture")
```

Comment:
I personally would start all these `doc=` strings with capitals. However, we aren't consistent about that across the codebase as it is, so I guess it's not a requirement.